### DISCUSSION TITLE
[4.0] Make it possible to choose the color of the Tags

### DISCUSSION THREAD

**Discussion by walturbo** [joomla/joomla-cms#47173] [2. Cold Features]
### Is your feature request related to a problem? Please describe.
Currently when creating a Tag there is no possibility to choose the color to give.

### Describe the solution you'd like
It could be a useful option for those who use several in the backend. When in an article you scroll the list of Tags at a glance you recognize the Tag that interests you regardless of what it contains. If instead you have different groups of Tags you can subdivide them by colors: for example on a recipe site you can group all the Tags that contain "vegetable ingredients" with a Green Tag, those that contain "Meat Ingredients" with a red Tag and etc.
They can also be useful in the frontend by showing them inside the articles. The user can navigate through the colored tags on the site and more easily find all the articles relating to a specific tag.

**Comment by coolcat-creations** [comment]
@walturbo you can color the tags with css and overrides. You can give each tag an own class in it's settings.

**Comment by walturbo** [comment]
@coolcat-creations If I'm not mistaken, in this way the tags will be colored only frontend, it would be useful to see them colored also in the backend in my opinion.

**Comment by jwaisner** [comment]
Setting as a feature request for J4 as J3 is not open for feature changes.<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/28600">issues.joomla.org/tracker/joomla-cms/28600</a>.</sub>

**Comment by alikon** [comment]
you can do the same i mean with a different override  to the backend too

**Comment by walturbo** [comment]
@alikon sorry but I'm not an expert! By overriding, can I change the color of the tags in the backend globally within the article, or am I wrong? In my opinion it would be interesting to get such a result, where I can decide the color for each Tag, sorry but maybe I had previously expressed myself wrong.

![git](https://user-images.githubusercontent.com/4063964/78796362-7ad4da00-79b6-11ea-89ff-bfc87e4b58cd.jpg)

**Comment by alikon** [comment]
well i'm not sure if your request should be a core feature at this point...but..
so let me reopen this to get more feedback....

**Comment by brianteeman** [comment]
In the admin the tags are created/displayed using a 3rd party script choices.js
It does not support individual styling. 

In the front end each tag has a unique class that can be used for styling it is tag-<tagid>
